.circleci/config.yml

```diff
@@ -16,31 +16,29 @@ jobs:
       - run: &setup_gcloud
           name: Setup gcloud
           command: |
-            echo ${GCLOUD_SERVICE_KEY} | base64 -d > ${HOME}/gcp-key.json
-            gcloud auth activate-service-account --key-file ${HOME}/gcp-key.json
+            echo ${GCLOUD_SERVICE_KEY} | base64 -d > ${HOME}/repo/gcp-key.json
+            gcloud auth activate-service-account --key-file ${HOME}/repo/gcp-key.json
             gcloud --quiet config set project model-159019
+            pwd
```

Comment:
Printing some test info to help debug the issues in circle

Reply:
I assume we can take these out when we're confident it's working correctly (and all the echos and whatnot)

Reply:
Yup! Just using them to try to understand why circle is unhappy 🙃